src/pkg/services/m365/api/helper_test.go

```diff
@@ -132,13 +132,14 @@ func newIntegrationTesterSetup(t *testing.T) intgTesterSetup {
 
 	its.siteDriveRootFolderID = ptr.Val(siteDriveRootFolder.GetId())
 
-	// teams
-	its.teamID = tconfig.M365TeamsID(t)
+	// group
 
-	team, err := its.ac.Groups().GetTeamByID(ctx, its.teamID)
+	its.groupID = tconfig.M365TeamID(t)
```

Comment:
```suggestion
	its.groupID = tconfig.M365GroupID(t)
```

Reply:
That's actually intentional.  I've added a comment to explain the choice.  In short: we want to ensure the group supports teams apis for full testing coverage.